slave.mk

```diff
@@ -230,7 +230,6 @@ $(info "INCLUDE_SFLOW"                   : "$(INCLUDE_SFLOW)")
 $(info "INCLUDE_NAT"                     : "$(INCLUDE_NAT)")
 $(info "INCLUDE_KUBERNETES"              : "$(INCLUDE_KUBERNETES)")
 $(info "TELEMETRY_WRITABLE"              : "$(TELEMETRY_WRITABLE)")
-$(info "ENABLE_SYNCHRONOUS_MODE"         : "$(ENABLE_SYNCHRONOUS_MODE)")
```

Comment:
>ENABLE_SYNCHRONOUS_MODE [](start = 8, length = 23)

Just wondering if there is a need to build for async/sync for community members.

Reply:
I feel it is not a must as long as we provide the community with the CLI to change the configuration. The users can update the mode after booting up the image, and this scenario should be pretty rare.